.gitignore

```diff
@@ -1 +1,9 @@
-_build
\ No newline at end of file
+_build
+
+# -------- CMake --------
+build/*
+
+# -------- IDE --------
+.vs/*

```

Comment:
Consider `.vscode/*` as well, it seems to be quite popular.

Reply:
Done

Reply:
cool, I've put the same comment but didn't press "Finish review" button

Reply:
Already done.